packages/mobile/src/qrcode/QRCode.tsx

```diff
@@ -29,12 +32,23 @@ export default function QRCodeDisplay({ qrSvgRef }: Props) {
     data.displayName,
     data.e164PhoneNumber,
   ])
+  const path = useMemo(() => pathFromUriData(data), [
+    data.address,
+    data.displayName,
+    data.e164PhoneNumber,
+  ])
+  const horizPadding = 40
+  const getQRSize = () => {
+    return Math.min((variables.width * 3) / 4, variables.width - horizPadding * 2)
+  }
+  const link = useAsync(() => generateLinkWithPath(path), [path])
```

Comment:
Is this going to generate new links everytime it's displayed?

Reply:
If you request a link with the same parameters it returns the same link from some my testing, so this shouldn't be an issue I think.